classes/class-wc-connect-taxjar-integration.php

```diff
@@ -1262,7 +1262,7 @@ public function validate_taxjar_request( $json ) {
 			return false;
 		}
 
-		if ( 'US' === $json['to_country'] && ! WC_Validation::is_postcode( $json['to_zip'], $json['to_country'] ) ) {
+		if ( 'US' === $json['to_country'] && ! empty( $json['to_zip'] ) && in_array( strlen( $json['to_zip'] ), array( 5, 10 ) ) && ! WC_Validation::is_postcode( $json['to_zip'], $json['to_country'] ) ) {
```

Comment:
Let's add a comment on what we're trying to do here. Perhaps something like : 
`// Only display the zip error if zip code length is 5 or 10.`